context/app/routes_main.py

```diff
@@ -58,6 +58,23 @@ def search():
     )
 
 
+@blueprint.route('/test-search/datasets')
+@blueprint.route('/test-search/donors')
+@blueprint.route('/test-search/samples')
+def test_search():
+    entity_type = request.path.split('/')[-1]
+    title = f'{entity_type.capitalize()}' if entity_type else 'Search'
```

Comment:
```suggestion
    title = f'{entity_type.capitalize()}'
```
I'm not sure there's a way to reach the else?